Models/rotors/fenestronanimation.xml

```diff
@@ -25,4 +25,43 @@
     <inherits-from>Effects/model-transparent</inherits-from>
     <object-name>rotor_disc_T</object-name>
   </effect>
+
+  <animation>
+    <!-- Rotate the fenestron blades -->
+    <type>rotate</type>
+    <object-name>Tblade</object-name>
+    <property>rotors/tail/blade/position-deg</property>
+    <factor>1</factor>
+    <center>
+      <x-m>0</x-m>
+      <y-m>0</y-m>
+      <z-m>0</z-m>
+    </center>
+    <axis>
+      <x>0</x>
+      <y>1</y>
+      <z>0</z>
+    </axis>
+  </animation>
+  <animation>
+    <!-- Hide the fenestron blades when rotating with high rpm -->
+    <type>select</type>
```

Comment:
Oje....

I never notice that the fenestron wasn't working.  I now took a look into the original repo at gitlab.com and discoverd the missing bits. It seems I simple forgot to paste them into the new branch I created 3years ago. I made a completely different approach without seperating the blades into a separate model. 

Hmm....